gradle/versions.gradle

```diff
@@ -57,6 +57,6 @@ dependencyManagement {
 
     dependency 'org.web3j:core:4.5.0'
     dependency 'org.web3j:crypto:4.5.0'
-    dependency 'org.web3j:pantheon:4.5.0'
+    dependency 'org.web3j:besu:4.5.2'
```

Comment:
Can we use the same version for all web3j modules

Reply:
done